releasenotes/notes/add-flexlayer-swap-pass-da827619e619faa1.yaml

```diff
@@ -0,0 +1,11 @@
+---
+features:
+  - |
+    Added a `FlexlayerSwap` pass, that implements a look-ahead heuristic
+    mapping algorithm. It can be used by replacing the `StochasticSwap`,
```

Comment:
I think it could be better to have a standalone example, using a physical circuit (e.i written in terms of `q` qubits) and show how it works. Otherwise, after #3004 gets merged (it will soon), you can replace the swapper using the new API. The table from this PR might be handy to show the improvement.